### DISCUSSION TITLE
Deploy Tutorial Video[ EDU EMAIL ]  ❤

### DISCUSSION THREAD

**Discussion by rinrinx2** [spencerwooo/onedrive-vercel-index#279] [Show and tell]
TUTORIAL

https://youtu.be/yzVtgaTqFD0

Exp Site https://kutuphanem.vercel.app

**Comment by rinrinx2** [comment]
@spencerwooo  bro pin this video to the beginning

**Reply by rinrinx2** [reply]
You made a mistake while deploying, there is a problem in a transaction
@Madhav-Sheth

**Reply by rinrinx2** [reply]
![Screenshot_20220115-221134_Chrome](https://user-images.githubusercontent.com/58888200/149634916-f1026853-31cf-444a-a1d9-fe258676ed24.jpg)


I look your site now worked

**Comment by shaojs321** [comment]
> 

Hi, brother, I was following your video production, and there was a problem in the last step

I have an educational mailbox in Onedrive

My site URL: https://tonyshao.vercel.app/

My Github submission :  https://github.com/shaojs321/onedrive-vercel-index

It doesn't work whether I turn on the proxy or not

@spencerwooo  Can you see what's wrong with me
<img width="831" alt="Snipaste_2022-01-18_02-49-24" src="https://user-images.githubusercontent.com/56724141/149824515-fb18a70e-69c8-4d18-8b5b-912596b82fc9.png">
<img width="1553" alt="Snipaste_2022-01-18_02-54-02" src="https://user-images.githubusercontent.com/56724141/149824527-998a2fd6-055f-4571-a628-61d962e113ae.png">

**Comment by shaojs321** [comment]
The picture uploaded just now may not be clear enough

![Snipaste_2022-01-18_02-59-26](https://user-images.githubusercontent.com/56724141/149824742-ed70a209-513a-4762-a52c-17e58ad3cc2c.png)

**Reply by shaojs321** [reply]
Hi, brother, I have watched the video again, it has been successful, thanks for your video
